### PR TITLE
Run e2e testnet tests via GHA, not Kubernetes

### PR DESCRIPTION
**Pull request by michalinacienciala**
We were running E2E tests on testnet (Ropsten) using Kubernetes. But as
the GitHub Actions provide better visibility for the tests results, we
decided to move the testing job there.
The `e2e-testnet.yml` has been created and the old Kubernetes config has
been removed.
The currently proposed solution is not ideal. It requires manual
updating of the `tbtc.js` version in the workflow config every time
new contracts get released. In the future we plan to do that
automatically.

### CODE REVIEW

File: .github/workflows/e2e-testnet.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: E2E tests / Testnet
+
+on:
+  schedule:
+    - cron: '*/20 * * * *'
+  workflow_dispatch:
+  pull_request: # TODO: remove this trigger before merging to main
```
Comment (nkuba) [inline .github/workflows/e2e-testnet.yml:7]: Can we remove it yet?

File: .github/workflows/e2e-testnet.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14.x"
+
+      # TODO: Temprorarily we use fixed version of tbtc.js. Ultimately, we would
+      # need to point to the latest deployed version.
+      - run: npm install --save-exact @keep-network/tbtc.js@0.19.4-ropsten.2
```
Comment (nkuba) [inline .github/workflows/e2e-testnet.yml:24]: Do we have a task in the backlog to cover that?

File: .github/workflows/e2e-testnet.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+      - name: Cache Global NPM Cache
```
Comment (nkuba) [inline .github/workflows/e2e-testnet.yml:26]: Do we need this step? Can't we use `actions/setup-node`'s cache?

File: .github/workflows/e2e-testnet.yml (hunk 2 above)
Comment (michalinacienciala) [reply .github/workflows/e2e-testnet.yml]: Added, https://keepnetwork.atlassian.net/browse/CP-56.

File: .github/workflows/e2e-testnet.yml (hunk 3 above)
Comment (michalinacienciala) [reply .github/workflows/e2e-testnet.yml:26]: We can't at this moment ([action's cache does not support `package.json` located in the repo's subfolder](https://github.com/actions/setup-node/issues/275)). But it looks that the fix for the issue is in the works, so maybe we will be able to do that in near future.

File: .github/workflows/e2e-testnet.yml (hunk 1 above)
Comment (michalinacienciala) [reply .github/workflows/e2e-testnet.yml:7]: We need to keep it as long as we need to test the workflow (cron and workflow_dispatch won't work until the workflow is merged to `main`). I've added the TODO in the PRs description in order not to forget about removing this once we're ready.

File: .github/workflows/e2e-testnet.yml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14.x"
+
+      # TODO: Temprorarily we use fixed version of tbtc.js. Ultimately, we would
+      # need to point to the latest deployed version.
+      - run: npm install --save-exact @keep-network/tbtc.js@0.19.4-ropsten.4
```
Comment (lukasz-zimnoch) [inline .github/workflows/e2e-testnet.yml:24]: Once https://github.com/keep-network/tbtc.js/pull/129 lands and a new package for `ropsten` is published, this line should be:
```
- run: npm install --save-exact @keep-network/tbtc.js@ropsten
```
and it should always pull the latest package for `ropsten`.

Comment (michalinacienciala) [reply .github/workflows/e2e-testnet.yml]: Added. After I published the `ropsten`-tagged `tbtc.js` package, step is passing, correct package was used.
Workflow run: [link](https://github.com/keep-network/local-setup/pull/98/checks?check_run_id=3202543359) (workflow is failing, but on later stage - will be investigated)

**Comment by lukasz-zimnoch** [comment]
Successful run: https://github.com/keep-network/local-setup/actions/runs/1158482530

File: .github/workflows/e2e-testnet.yml (hunk 1 above)
Comment (lukasz-zimnoch) [reply .github/workflows/e2e-testnet.yml]: Replaced it with a cron which runs this job four times a day: https://github.com/keep-network/local-setup/pull/98/commits/152e9f00514a7066d646d0c4bf72f36c61df2e77

File: .github/workflows/e2e-testnet.yml (hunk 3 above)
Comment (lukasz-zimnoch) [reply .github/workflows/e2e-testnet.yml]: Looks we already have this one: https://github.com/keep-network/local-setup/pull/98/commits/724cca168ea801db494325fc5457f6ccea1e68d3

**Review by nkuba — APPROVED** [review]

**Pull request merged**